### PR TITLE
Proj0030: Use VB.NET specific properties only in when applicable

### PR DESCRIPTION
**Pull request by Corniel**
Documentation for https://github.com/dotnet-project-file-analyzers/dotnet-project-file-analyzers/pull/223.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ reported a the [GibHub repository](https://github.com/dotnet-project-file-analyz
 * [**Proj0026** Remove IncludeAssets when redundant](rules/Proj0026.md)
 * [**Proj0027** Override &lt;TargetFrameworks&gt; with &lt;TargetFrameworks&gt;](rules/Proj0027.md)
 * [**Proj0028** Define conditions on level 1](rules/Proj0028.md)
+* [**Proj0030** Use VB.NET specific properties only in when applicable](rules/Proj0030.md)
```
Comment (Gemberkoekje) [inline README.md:70]:
```suggestion
* [**Proj0030** Use VB.NET specific properties only when applicable](rules/Proj0030.md)
```

File: rules/Proj0030.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Proj0030: Use VB.NET specific properties only in when applicable
```
Comment (Gemberkoekje) [inline rules/Proj0030.md:1]:
```suggestion
# Proj0030: Use VB.NET specific properties only when applicable
```

File: rules/Proj0030.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+When building a VB.NET project, MS Build supports multiple VB.NET only
+properties to be set. For none VB.NET projects however, settings these
```
Comment (Gemberkoekje) [inline rules/Proj0030.md:3]:
```suggestion
properties to be set. For none VB.NET projects however, setting these
```

**Review by Gemberkoekje — COMMENTED** [review]
Couple of typo's

**Review by LauraXiulan — APPROVED** [review]
Approved with @Gemberkoekje's suggestions

**Pull request merged**